skorch/setter.py

```diff
@@ -0,0 +1,42 @@
+"""Setter functions for virtual params such as ``optimizer__lr``."""
+import re
+
+from skorch.utils import set_optimizer_param
+
+
+def optimizer_setter(
+        net, param, value, optimizer_attr='optimizer_', optimizer_name='optimizer'
```

Comment:
When would we set other values for `optimizer_attr` and `optimizer_name`? Perhaps a docstring complete with parameter description would help here.